examples/quickstart/jupyter-notebooks/druidapi/setup.py

```diff
@@ -0,0 +1,37 @@
+# Licensed to the Apache Software Foundation (ASF) under one or more
+# contributor license agreements.  See the NOTICE file distributed with
+# this work for additional information regarding copyright ownership.
+# The ASF licenses this file to You under the Apache License, Version 2.0
+# (the "License"); you may not use this file except in compliance with
+# the License.  You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+from setuptools import setup, find_packages
+
+setup(
+    name='druidapi',
+    version='0.1.0',
+    description='Python API client for Apache Druid',
+    url='https://github.com/apache/druid/tree/master/examples/quickstart/jupyter-notebooks/druidapi',
+    author='Apache Druid project',
+    author_email='dev@druid.apache.org',
+    license='Apache License 2.0',
+    packages=find_packages(),
+    install_requires=['requests'],
```

Comment:
One thing to consider when we have more dependencies is to add a separate setup target with the other manually installed dependencies like `jupyterlab`/`notebook`.
Similarly for the [project build dependency](https://druid.apache.org/docs/latest/development/build.html) that requires `pyyaml`.